buycex/chain.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "buycex",
+  "chain_id": "bcx_323-1",
+  "pretty_name": "BuyCex Infinity Chain",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://infinity.buycex.com",
+  "bech32_prefix": "bcx",
+  "daemon_name": "bcxd",
+  "node_home": "$HOME/.bcxd",
+  "chain_type": "cosmos-sdk",
```

Comment:
Per the schema, cosmos-sdk is not an accepted chain type, although "cosmos" is allowed